tests/handlers/botas/test_botas_shipped_advisory.py

```diff
@@ -207,3 +232,47 @@ def test_filter_bundles_by_pinned_related_images(self, service):
         temp_mock.get_build.assert_has_calls([call(bundle_list[0]),
                                              call(bundle_list[1])])
         self.assertEqual(nvrs, {bundle_list[0]})
+
+    def test_no_original_nvr_found(self):
+        event1 = Event.create(db.session, "id1", "RHSA-1", TestingEvent)
+        ArtifactBuild.create(db.session, event1, "ed0", "image", 1234,
+                             original_nvr="nvr1-0-1",
+                             rebuilt_nvr="nvr1-0-2")
+        db.session.commit()
+        self.pyxis()._pagination.return_value = [
+            {"repositories": [{"published": True}]}
+        ]
+
+        ret_nvr = self.handler.get_published_original_nvr(db.session, "nvr1-0-2")
+        self.assertEqual(ret_nvr, 'nvr1-0-1')
+
+    def test_get_published_original_nvr(self):
```

Comment:
To make the test better, could you assign each build to a separate event? Bonus points if you use the `ManualRebuildWithAdvisoryEvent` event type for the last two builds.

Reply:
Previously I had it like that, but I thought that since I don't query Events table at all there is no need in separate events.